torchbenchmark/models/hf_T5/__init__.py

```diff
@@ -23,22 +21,25 @@ def forward(self, input_ids, decoder_input_ids):
 class Model(BenchmarkModel):
     task = NLP.LANGUAGE_MODELING
 
-    def __init__(self, device=None, jit=False):
+    # Original train batch size per device: 8
+    # Source: https://github.com/huggingface/transformers/blob/master/examples/flax/language-modeling/run_t5_mlm_flax.py#L83
+    # Original eval batch size per device: 8
+    # Downscale to 1 to fit in Nvidia T4 of the infra
+    def __init__(self, device=None, jit=False, train_bs=8, eval_bs=1):
         super().__init__()
         self.device = device
         self.jit = jit
 
-        torch.manual_seed(42)
         config = AutoConfig.from_pretrained("t5-small")
         self.model = AutoModelForSeq2SeqLM.from_config(config).to(device)
         self.optimizer = optim.Adam(self.model.parameters(), lr=0.001)
 
-        input_ids = torch.randint(0, config.vocab_size, (2, 1024)).to(device)
-        decoder_ids = torch.randint(0, config.vocab_size, (2, 1024)).to(device)
+        # input_ids = torch.randint(0, config.vocab_size, (train_bs, 1024)).to(device)
+        # decoder_ids = torch.randint(0, config.vocab_size, (train_bs, 1024)).to(device)
```

Comment:
Should we delete the commented out parts?

Reply:
We can keep it there so it is easier to re-enable the train